filestore/test/test_write.py

```diff
@@ -65,14 +64,14 @@
 conn = None
 
 
-def setup():
+def setup_module(module):
     fs_setup()
```

Comment:
:bike::house: Why not use a pytest fixture for this? Switching to py.test, may as well use its features